File: Web/AppServer/AppServer/Models/SignalDirection.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace AppServer.Models
{
    /// <summary>
    /// Defines signal type, if it is a signal coming from or to device
    /// </summary>
    public enum SignalDirection
    {
        ToDevice = 1,
        FromDevice = 2
    }
}

File: Web/appserver/src/main/java/com/appserver/models/SignalDirection.java
package com.appserver.models;

import com.google.gson.annotations.SerializedName;

/**
 * Defines signal type, if it is a signal coming from or to device
 */
public enum SignalDirection {
   	/**
   	 * Signal coming to device
   	 */
	@SerializedName("1")
	ToDevice(1),
    /**
     * Signal coming from device
     */
	@SerializedName("2")
    FromDevice(2);
    
    private final int value;
    private SignalDirection(int value) {
        this.value = value;
    }

    public int getValue() {
        return value;
    }
}
